### PR TITLE
In-2527 summary deal info list create update

### PR DESCRIPTION
**Pull request by EOSullivanBerlin**

### CODE REVIEW

**Review by lowiebenoot — DISMISSED** [review]

**Review by sanderbrugge — DISMISSED** [review]

**Review by luisaterrabrito — DISMISSED** [review]

File: src/03-deals/deals.apib [hunk 1]
```diff
@@ -77,6 +77,7 @@ Get a list of deals.
                 + estimated_value (Money)
                 + estimated_closing_date: `2017-05-09` (string)
                 + estimated_probability: `0.5` (number)
+                + summary: `Additional information` (string, nullable)
```
Comment (dietrichm) [inline src/03-deals/deals.apib]: PP: to me this goes hand in hand with the title, so I think it should go after `title` or at least closer to the top of the properties.

File: src/03-deals/deals.apib [hunk 2]
```diff
@@ -130,6 +131,7 @@ Get details for a single deal.
             + estimated_closing_date: `2017-05-09` (string)
             + estimated_probability: `0.5` (number)
             + weighted_value (Money)
+            + summary: `Additional information` (string, nullable)
```
Comment (dietrichm) [inline src/03-deals/deals.apib]: PP: same here.

File: src/03-deals/deals.apib [hunk 3]
```diff
@@ -182,6 +184,7 @@ Create a new deal for a customer.
         + estimated_probability: `0.75` (number, optional) - A number between 0 and 1 (inclusive)
         + estimated_closing_date: `2017-05-09` (string, optional)
         + custom_fields (array[CustomFieldValue], optional)
+        + summary: `Additional information` (string, optional)
```
Comment (dietrichm) [inline src/03-deals/deals.apib]: PP: same here.

File: src/03-deals/deals.apib [hunk 4]
```diff
@@ -214,6 +217,7 @@ Update a deal.
         + estimated_probability: `0.75` (number, optional, nullable) - A number between 0 and 1 (inclusive)
         + estimated_closing_date: `2017-05-09` (string, optional, nullable)
         + custom_fields (array[CustomFieldValue], optional)
+        + summary: `Additional information` (string, optional)
```
Comment (dietrichm) [inline src/03-deals/deals.apib]: PP: same here.

File: src/03-deals/deals.apib (hunk 1 above)
Comment (EOSullivanBerlin) [reply src/03-deals/deals.apib]: Alright I can move it up.

**Review by sanderbrugge — DISMISSED** [review]

File: src/03-deals/deals.apib [hunk 5]
```diff
@@ -219,6 +222,7 @@ Update a deal.
                 + id: `09e5d75f-f817-4872-9723-57fbb8ff710d` (string, required)
             + contact_person_id: `b38ebb9b-6e46-4bf4-a1e2-af747d6b64ae` (string, optional)
         + title: `Interesting business deal` (string, optional)
+        + summary: `Additional information` (string, optional)
```
Comment (dietrichm) [inline src/03-deals/deals.apib]: Q: how can an integrator clear the summary for an existing deal?

Comment (EOSullivanBerlin) [reply src/03-deals/deals.apib]: Good point so I guess that I need to allow empty strings after all.

File: apiary.apib [hunk 6]
```diff
@@ -1820,6 +1825,7 @@ Update a deal.
                 + id: `09e5d75f-f817-4872-9723-57fbb8ff710d` (string, required)
             + contact_person_id: `b38ebb9b-6e46-4bf4-a1e2-af747d6b64ae` (string, optional)
         + title: `Interesting business deal` (string, optional)
+        + summary: `Additional information` (string, optional)
```
Comment (dietrichm) [inline apiary.apib]: B: main Apiary file has not been rebuilt yet.

**Review by dietrichm — DISMISSED** [review]

**Review by lowiebenoot — APPROVED** [review]

**Review by sanderbrugge — APPROVED** [review]

**Review by dietrichm — APPROVED** [review]

**Review by steftrenson — APPROVED** [review]

**Review by luisaterrabrito — APPROVED** [review]

**Pull request merged**